deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -727,7 +727,7 @@ groups:
     rules:
       - alert: CephPoolGrowthWarning
         expr: |
-          (predict_linear(ceph_pool_percent_used[2d], 3600 * 24 * 5) * on(pool_id)
+          (predict_linear((max(ceph_pool_percent_used) without (pod, instance))[2d:1h], 3600 * 24 * 5) * on(pool_id)
```

Comment:
might want to make the changes [here](https://github.com/rook/rook/blob/56b28df5f1fa2e891afb597a15824cb4dabf2a06/deploy/examples/monitoring/localrules.yaml#L738) as well.

Reply:
done